.github/workflows/update-compat-data.yml

```diff
@@ -49,7 +49,9 @@ jobs:
       - name: Update tests (Babel 8)
         if: steps.updated.outputs.updated == 'true'
         continue-on-error: true
-        run: yarn jest
+        run: |
+          make use-esm
+          yarn jest
```

Comment:
The recent workflows failed because of too many test errors: 

https://github.com/babel/babel/actions/runs/6115942789